ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/EthHashMiningCoordinator.java

```diff
@@ -49,6 +55,18 @@ public void setStratumMiningEnabled(final boolean stratumMiningEnabled) {
     executor.setStratumMiningEnabled(stratumMiningEnabled);
   }
 
+  @Override
```

Comment:
nit: could/should this stay in the base class? and just overriden in clique?

Reply:
I have just changed; I reduced to minimum what must be overriden

Reply:
looks way better (IMHO)